docs/buildah-bud.md

```diff
@@ -1,28 +1,29 @@
 # buildah-bud "1" "April 2017" "buildah"
 
 ## NAME
-buildah\-bud - Build an image using instructions from Dockerfiles.
+buildah\-bud - Build an image using instructions from Containerfiles
 
 ## SYNOPSIS
+
 **buildah build-using-dockerfile** [*options*] [*context*]
 
 **buildah bud** [*options*] [*context*]
 
 **bud** is an alias for **build-using-dockerfile**.
 
 ## DESCRIPTION
-Builds an image using instructions from one or more Dockerfiles and a specified
-build context directory.
+Builds an image using instructions from one or more Containerfiles or Dockerfiles and a specified
+build context directory.  A Containerfile uses the same syntax as a Dockerfile internally.
 
-The build context directory can be specified as the http(s) URL of an archive, git repository or Dockerfile.
+The build context directory can be specified as the http(s) URL of an archive, git repository or Containerfile.
 
-If no context directory is specified, then buildah will assume the current working directory as build context, which should contain the Dockerfile.`
+If no context directory is specified, then Buildah will assume the current working directory as build context, which should contain either a Containerfile or a Dockerfile.`
 
 Dockerfiles ending with a ".in" suffix will be preprocessed via CPP(1).  This can be useful to decompose Dockerfiles into several reusable parts that can be used via CPP's **#include** directive.  Notice, a Dockerfile.in file can still be used by other tools when manually preprocessing them via `cpp -E`.
 
 When the URL is an archive, the contents of the URL is downloaded to a temporary location and extracted before execution.
 
-When the URL is an Dockerfile, the Dockerfile is downloaded to a temporary location.
+When the URL is a Dockerfile, the Dockerfile is downloaded to a temporary location.
```

Comment:
Will this work with a Containerfile as well?

Reply:
I'm not sure and didn't test it so didn't want to be bold.  I'll see if I can work up a test.